app/xlist/scripts/HomeController.js

```diff
@@ -124,9 +124,28 @@ angular
             });
           });
         }).then(function() {
-          $scope.pairs = _.sortBy(newPairs, function(pair) {
-            return pair.geoList.name;
-          });
+          if (newPairs.length === 0) {
+            var newGeoPoint = new ParseObject(new GeoList(), GeoList.fields);
+            newGeoPoint.name = 'To-Do List';
+            newGeoPoint.location = new Parse.GeoPoint({
+              latitude: 0.0,
```

Comment:
Maybe not for this PR, but do we want to change the LocationController behavior to default to user's location when no location is set? This might be something to tackle in multiple lists.